src/App.tsx

```diff
@@ -1,15 +1,45 @@
 /* eslint-disable max-len */
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 
 import { TodoList } from './components/TodoList';
 import { TodoFilter } from './components/TodoFilter';
 import { TodoModal } from './components/TodoModal';
+import { getTodos } from './api';
+import { Todo } from './types/Todo';
 import { Loader } from './components/Loader';
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([])
+  const [loading, setLoading] = useState(true);
+  const [initialTodos, setInitialTodos] = useState<Todo[]>([])
+  const [active, setActive] = useState(false)
+  const [selectedUserId, setSelectedUserId] = useState<number | null>(null);
+  const [selectedTodo, setSelectedTodo] = useState<Todo | null>(null);
+
+  useEffect(() => {
+    getTodos().then((data) => {
+      setTodos(data)
+      setInitialTodos(data)
+    })
+      .finally(() => setLoading(false))
```

Comment:
Consider adding error handling for the `getTodos` API call. This will help manage any potential errors that might occur during the fetching process. You can use a `.catch()` block to handle errors appropriately.